networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -108,14 +109,14 @@ def bfs_predecessors(G, source):
 
     Returns
     -------
-    pred: dict
-       A dictionary with nodes as keys and predecessor nodes as values.
+    pred: iterator
+        (key, predecessor) iterator
```

Comment:
should the word "key" here (in doc_string) be "node"? 
Doesn't really matter but perhaps more clear? Same in other doc_strings that use "key"...